openvdb/openvdb/python/test/TestOpenVDB.py

```diff
@@ -60,6 +84,46 @@ def testModule(self):
             self.assertEqual(grid.activeVoxelCount(), 1)
 
 
+    def testAX(self):
+        if not ax_is_enabled():
+            return
+
+        float_grid = openvdb.FloatGrid()
+        float_grid.name = 'float_grid'
+        ijk = (1, 1, 1)
+
+        acc = float_grid.getAccessor()
+        acc.setValueOn((ijk))
+        self.assertEqual(acc.getValue(ijk), 0)
+
+        openvdb.ax('@float_grid = 2;', float_grid)
+        self.assertEqual(acc.getValue(ijk), 2)
+        acc.setValueOn((ijk))
+
+        float_grid.fill((0, 0, 0), (7, 7, 7), -1, active=True)
+        openvdb.ax('@float_grid = lengthsq(getcoord());', float_grid)
+
+        for i in range(0, 8):
+            for j in range(0, 8):
+                for k in range(0, 8):
+                    ijk = (i, j, k)
+                    lsq = (i*i)+(j*j)+(k*k)
+                    self.assertEqual(acc.getValue(ijk), lsq)
+
+        vec3_grid = openvdb.Vec3SGrid()
+        vec3_grid.name = 'vec_grid'
+
+        vec3_grid.fill((0, 0, 0), (7, 7, 7), (-1,-1,-1), active=True)
+        openvdb.ax('v@vec_grid = @float_grid;', [vec3_grid, float_grid])
+
+        for i in range(0, 8):
+            for j in range(0, 8):
+                for k in range(0, 8):
+                    ijk = (i, j, k)
+                    lsq = (i*i)+(j*j)+(k*k)
+                    self.assertEqual(acc.getValue(ijk), float(lsq))
```

Comment:
should this be checking the vec3_grid values?

Reply:
Oh great catch, will fix

Reply:
Fixed